### DISCUSSION TITLE
I still havnt gotten the email for the verification

### DISCUSSION THREAD

**Discussion by BISHER-AL-MASRI** [community/community#69332] [GitHub Education]
### Hi everyone,

Long story short

I applied for student benifets and sitll havnt gotten accepted it showen it is purple and its been aproved but still no email.

<img width="1373" alt="image" src="https://github.com/community/community/assets/106201955/d920aed1-013a-4f02-97b2-aae892d63f3c">

<img width="1048" alt="image" src="https://github.com/community/community/assets/106201955/b21f49ba-babe-4c3b-a391-6dab63329a4e">

**Comment by mantasu** [comment]
I think you're having the same issue as some other people. See the answer [here](https://github.com/orgs/community/discussions/69264)

Basically, you may still need to wait for a few more days as it seems the staff is super busy currently. It doesn't hurt to open a [ticket](https://support.github.com/request/education), just remember to close it once your benefits arrive.
